sql/teradata/README.md

```diff
@@ -0,0 +1,31 @@
+# Teradata SQL Grammar  
+
+An [ANTLR4](https://www.antlr.org/) grammar for Teradata SQL. Based on a grammar of Teradata Database version 17.10.  
```

Comment:
Please include a permalink to the actual grammar for version 17.10. Is it a bison grammar? Is there a lex file?

Reply:
Any of these?
* https://docs.teradata.com/r/Enterprise_IntelliFlex_VMware/SQL-Fundamentals/Basic-SQL-Syntax/SQL-Statement-Structure. The problem with this link is that the version "floats". There's no *fixed* version of the SQL syntax here. People can sneak in a change without anyone detecting the change.
* Is there anything under any of these Github repos to a pdf? https://github.com/Teradata
* Is there a PDF that contains text stating "version 17.10 of the SQL">
* Does wayback.archive.org have a link to the doc fixed in time?

Reply:
I've added links to 17.10 docs. Those docs URLs end with `/July-2021`  subpath.